query-connector/src/app/queryBuilding/buildFromTemplates/BuildFromTemplates.tsx

```diff
@@ -88,13 +117,6 @@ export default function QueryTemplateSelection() {
       const formattedResults =
         results && groupConditionConceptsIntoValueSets(results);
 
-      // when fetching directly from conditions table (as opposed to a saved query),
```

Comment:
I went ahead and changed the default return from the formatting function to have the `include` values to be true, since in any case that I can think of, fetching a valueset / concept means that we want to use it. Lmk if this is an incorrect assumption though.

Reply:
I think that's right for a default condition/valueset/concept tree that we'd include everything. I also like the code to deselect a valuset if someone deselects all concepts.